src/analytics.ts

```diff
@@ -30,16 +30,24 @@ export function createAnalyticsInstance(options?: Options) {
         user_id,
         app_id,
     }: TCoreAttributes) => {
+        const attributes = {
+            ...(country !== undefined && { country }),
+            ...(user_language !== undefined && { user_language }),
+            ...(device_language !== undefined && { device_language }),
+            ...(device_type !== undefined && { device_type }),
```

Comment:
```suggestion
            ...(country !== undefined ? { country }: {}),
            ...(user_language !== undefined ? { user_language }: {}),
```

isn't this should be like this ? performing a spread operation `...` on boolean value could throw an error

Reply:
if some property is undefined, the param will not copy at all :) 
example is proof

Reply:
Sounds good, I thought running spread operator on boolean will throw the error, but it turns out that it won't 😄